src/Domain.jl

```diff
@@ -1577,6 +1577,29 @@ end
 end
 export calcdomainderivatives!
 
+function jacobiany!(jac::Q,y::U,p::W,t::Z,domain::D,interfaces::Q3,colorvec::Q2=nothing) where {Q3<:AbstractArray,Q2,Q<:AbstractArray,U<:AbstractArray,W,Z<:Real,D<:ConstantTPDomain}
+    ns,cs,T,P,V,C,N,mu,kfs,krevs,Hs,Us,Gs,diffs,Cvave = calcthermo(domain,y,t,p)
+    jacobiany_ns!(jac,y,p,t,domain,[],cs,kfs,krevs,nothing)
+    for i in domain.indexes[1]:domain.indexes[2]
+        jac[domain.indexes[3],i] = sum(jac[domain.indexes[1]:domain.indexes[2],i])*R*T/P
+    end
+    
+    for inter in interfaces
+        if isa(inter,Outlet) && domain == inter.domain
```

Comment:
Aren't we missing the contribution of the outlet to the reactor volume here?

Reply:
> Aren't we missing the contribution of the outlet to the reactor volume here?

Since we use molar flow to specify the outlet, the contribution from the outlet flow is dV/dt -= inter.F(t)*R*T/P, so d/dni dV/dt -= 0